packages/xs-extra/wsproxy.master/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+name: "wsproxy"
+version: "1.1.0"
+authors: [ "Jon Ludlam" "jonathan.ludlam@citrix.com"
+           "Marcello Seri <marcello.seri@citrix.com>" ]
```

Comment:
`opam lint` says that `maintainer:` should be defined, as it's mandatory, maybe it could be `maintainer: "xen-api@lists.xen.org"`?